cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -166,10 +189,21 @@ ensure_caBundle "mutatingwebhookconfiguration"   "network-resources-injector-con
 envsubst < $MANIFESTS_DIR/network_config_policy.yaml | _kubectl create -f -
 
 sriov_operator_namespace="sriov-network-operator"
+sriov_cni_label="app=sriov-cni"
+sriov_device_plugin_label="app=sriov-device-plugin"
 
 # Ensure SriovNetworkNodePolicy CR is created
 policy_name=$(cat $MANIFESTS_DIR/network_config_policy.yaml | grep 'name:' | awk '{print $2}')
 ensure_cr "SriovNetworkNodePolicy" $policy_name $sriov_operator_namespace
 
+# Wait for sriov-operator to reconsile SriovNodeNetworkPolicy
```

Comment:
This comment does not really describe what is done on following lines. It just waits for pods to be created. Could we do that sooner? Right deploying the operator? And fix the comment

Reply:
I elaborated on the commit message, should I document it in the code?
`sriov-cn` and `sriov-device-plugin` pods are rescheduled even though there are already on a node
that should be affected by applying a policy, so its not enough to check it right after operator deployment.

Reply:
ah, makes sense